batchservice/src/main/java/nu/yona/server/batch/quartz/JobController.java

```diff
@@ -0,0 +1,148 @@
+package nu.yona.server.batch.quartz;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.ExposesResourceFor;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.hateoas.mvc.ControllerLinkBuilder;
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import nu.yona.server.rest.JsonRootRelProvider;
+
+@Controller
+@ExposesResourceFor(JobDto.class)
+@RequestMapping(value = "/scheduler/jobs", produces = { MediaType.APPLICATION_JSON_VALUE })
+public class JobController
+{
+	@Autowired
+	private JobManagementService jobManagementService;
+
+	@RequestMapping(value = "/", method = RequestMethod.GET)
+	@ResponseBody
+	public HttpEntity<Resources<JobResource>> getAllJobs()
+	{
+		return new ResponseEntity<>(
+				createJobsCollectionResource(jobManagementService.getAllJobs(), getAllJobsLinkBuilder().withSelfRel()),
+				HttpStatus.OK);
+	}
+
+	@RequestMapping(value = "/{group}/", method = RequestMethod.GET)
+	@ResponseBody
+	public HttpEntity<Resources<JobResource>> getJobsInGroup(@PathVariable String group)
+	{
+		return new ResponseEntity<>(createJobsCollectionResource(jobManagementService.getJobsInGroup(group),
+				getJobsInGroupLinkBuilder(group).withSelfRel()), HttpStatus.OK);
+	}
+
+	@RequestMapping(value = "/{group}/", method = RequestMethod.PUT)
```

Comment:
True. Seen from the back, it would have been better to start with just one group. However, I expect we will see a number of jobs for various purposes. That's why I went down this path. Now that it's there, don't like throwing it away.